hw/dv/tools/dvsim/riviera.hjson

```diff
@@ -9,12 +9,12 @@
   // {tool_srcs_dir} before running the simulation.
   tool_srcs:  ["{dv_root}/tools/riviera/*"]
 
-  build_opts: ["-sv",
-               "-timescale 1ns/1ps",
-               "-uvmver 1.2",
+  build_opts: ["-timescale 1ns/1ps",
+               "+incdir+\"{ALDEC_PATH}/vlib/uvm-1.2/src\"",
+               "\"{ALDEC_PATH}/vlib/uvm-1.2/src/uvm_pkg.sv\"",
                "-f {sv_flist}",
                // List multiple tops for the simulation. Prepend each top level with `-top`.
-               "{eval_cmd} echo {sim_tops} | sed -E 's/(\\S+)/-top \\1/g'",
+               //"{eval_cmd} echo {sim_tops} | sed -E 's/(\\S+)/-top \\1/g'",
```

Comment:
Doesn't Riviera need to know what top level designs to elaborate? I suggest keeping this and debugging the value of `{sim_tops}` in the icache dv env instead.